src/Octoshift/Services/ReclaimService.cs

```diff
@@ -177,9 +181,22 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
 
             if (skipInvitation)
             {
-                //TODO: Check if org is emu before continuing, throw error if not
+                // Check if user is admin to EMU org
+                var membership = await _githubApi.GetOrgMembershipForUser(githubTargetOrg, claimantLogin);
+
+                if (membership != "admin")
+                {
+                    _log.LogError($"User {claimantLogin} is not an org admin and is not eligible to reclaim the {login} mannequin. Skipping.");
+                    continue;
+                }
```

Comment:
does this mean if we are reclaiming 100 mannequins, all 100 users need to be org admins?

Reply:
That was how I interpreted the [issue](https://github.com/github/octoshift/issues/6913), @brianaj do you have clarification on this?

Reply:
> User not admin to EMU org

I interpreted `user` as the person running the CLI command to reclaim mannequins. Claimant users are just regular users in the EMU that we're attributing data to and are likely not admins in most cases.

I'm not sure if there's a way to check admin status for the user submitting requests without knowing the login information of the user. Maybe we can issue request to the viewer gql query (if if there's a REST api equivalent) to fetch login of the person submitting the request and then check against that?

```
query{
  viewer{
    login
  }
} 
```

```
{
	"data": {
		"viewer": {
			"login": "dpmex4527"
		}
	}
}
```

Reply:
Yes as @dpmex4527 only the user running query needs admin access. Similar to create attribution inviations.

Reply:
Addressed in 0c220ef6de8422b03b8f6ca43be1e154b7775c3f :tada: